validator/client/beacon-api/BUILD.bazel

```diff
@@ -4,12 +4,19 @@ load("@prysm//tools/go:def.bzl", "go_library")
 # keep
```

Comment:
FYI you can add this here and it will fix the problem:

```
# gazelle:build_tag use_beacon_api
```

Reply:
Fixed upstream